src/pages/state/[state].js

```diff
@@ -224,6 +210,20 @@ export default function State({ data: { statesGeoData }, state = "DC" }) {
             state={stateData.abbreviation}
           />
           <EquitySection size="sm" />
+          {stateData.videos?.length ? (
+            <Row className={styles.voices}>
+              <Row>
+                <h3>Voices For Reform</h3>
+              </Row>
+              {stateData.videos.map(({ link, title, description }, i) => (
+                <Row key={`statewide-videos-${i}`}>
+                  <ReactPlayer url={link} width={594} height={361} />
+                  <h4>{title}</h4>
+                  {description && <p>{description}</p>}
```

Comment:
Not in the original ticket, but this `description` should actually be wrapped in `ReactMarkdown` instead of `p`

_In_ the original ticket: can we un-bold this description? The transition to markdown might cover this? Otherwise there might be a stylesheet rule we have to undo..